docker-compose.yaml

```diff
@@ -22,6 +22,7 @@ services:
       - ${PWD}:/app
       - ${PWD}/scratch:/scratch
     environment:
+      - GLOBAL_CONFIG_URL=https://raw.githubusercontent.com/kbase/index_runner_spec/36780d4549e22b5f68bfac53cf6666a50c7c62c1/config.yaml
```

Comment:
This is temporary, just for testing. We need index names with underscores instead of colons

Reply:
ah, ignore comment below.